src/query/tests/RTreeTests.cpp

```diff
@@ -101,7 +101,6 @@ BOOST_AUTO_TEST_CASE(Query2DVertex)
 
   auto result = indexTree.getClosestVertex(location);
   BOOST_TEST(mesh->vertices().at(result.index).getCoords() == Eigen::Vector2d(0, 1));
-  BOOST_TEST(result.distance == 0.28284271247461906);
```

Comment:
In principle, these tests should now move to the polation tests, right?

Reply:
The interface is still returning a `ProjectionMatch` which is Polation + distance.

We really need to refactor the interface to return the Polation only, as it contains the distance. Then we should redistribute the test.

I didn't do this to get a solution quickly. I'll open an issue to do this after the release.

Reply:
I will leave it up to you. I could imagine that it would be easier and faster to do it now, where you are already into the topic, but opening an issue is also fine.

Reply:
The mentioned test doesn't check the distance, as getClosestVertex doesn't return the distance any longer. Got muddled up with the projection tests.

I am still going to for the refactoring.